docs/python/requirements.txt

```diff
@@ -9,3 +9,7 @@ sphinx_rtd_theme
 pyquickhelper
 pandas
 pydot
+flatbuffers
```

Comment:
Requirements.txt seems to be sorted. It would be good to insert the new requirements in alphabetical order.